experimental/ComplexReflectionGroups/src/complex_reflection_group_CHEVIE.jl

```diff
@@ -0,0 +1,609 @@
+        s3 = matspace(K[0 1//7*(-z_7^4 - z_7^2 - z_7 - 4) 1//4*(-z_7^4 - z_7^2 - z_7 - 1) ; 1//3*(z_7^4 + z_7^2 + z_7 - 3) 1//3 1//12*(-3*z_7^4 - 3*z_7^2 - 3*z_7 - 5) ; 1//3*(2*z_7^4 + 2*z_7^2 + 2*z_7) 1//21*(6*z_7^4 + 6*z_7^2 + 6*z_7 - 4) 2//3])
+                            
+        push!(gens, s1)
+        push!(gens, s2)
+        push!(gens, s3)
+
+        W = matrix_group(gens)
+
+    elseif n == 25
+        K,z_3 = cyclotomic_field(3)
+        matspace = matrix_space(K, 3, 3)
+        gens = elem_type(matspace)[]
+
+        s1 = matspace(K[1 0 0 ; 0 1 0 ; 0 0 z_3])
+        s2 = matspace(K[1//3*(z_3 + 2) 1//3*(z_3 - 1) 1//3*(z_3 - 1) ; 1//3*(z_3 - 1) 1//3*(z_3 + 2) 1//3*(z_3 - 1) ; 1//3*(z_3 - 1) 1//3*(z_3 - 1) 1//3*(z_3 + 2)])
+        s3 = matspace(K[1 0 0 ; 0 z_3 0 ; 0 0 1])
+                            
+        push!(gens, s1)
+        push!(gens, s2)
+        push!(gens, s3)
+
+        W = matrix_group(gens)
+
+    elseif n == 26
+        K,z_3 = cyclotomic_field(3)
+        matspace = matrix_space(K, 3, 3)
+        gens = elem_type(matspace)[]
+
+        s1 = matspace(K[1 0 0 ; 0 0 1 ; 0 1 0])
+        s2 = matspace(K[1 0 0 ; 0 1 0 ; 0 0 z_3])
+        s3 = matspace(K[1//3*(z_3 + 2) 1//3*(z_3 - 1) 1//3*(z_3 - 1) ; 1//3*(z_3 - 1) 1//3*(z_3 + 2) 1//3*(z_3 - 1) ; 1//3*(z_3 - 1) 1//3*(z_3 - 1) 1//3*(z_3 + 2)])
+                            
+        push!(gens, s1)
+        push!(gens, s2)
+        push!(gens, s3)
+
+        W = matrix_group(gens)
+
+    elseif n == 27
+        K,z_15 = cyclotomic_field(15)
+        matspace = matrix_space(K, 3, 3)
+        gens = elem_type(matspace)[]
+
+        s1 = matspace(K[1//2 1//10*(-2*z_15^7 + 2*z_15^6 + 2*z_15^5 - 6*z_15^4 + 4*z_15^3 + 2*z_15^2 + 2*z_15 + 3) 1//10*(-7*z_15^7 + 2*z_15^6 + 2*z_15^5 - z_15^4 - z_15^3 - 3*z_15^2 - 3*z_15 + 3) ; 1//10*(-8*z_15^7 + 6*z_15^6 + 2*z_15^5 - 6*z_15^4 + 2*z_15^3 - 4*z_15^2 + 2*z_15 + 5) 1//10*(2*z_15^7 - 2*z_15^3 + 2*z_15^2 - 1) 1//10*(7*z_15^7 + 5*z_15^4 - 7*z_15^3 + 7*z_15^2 + 5*z_15 - 1) ; 1//15*(2*z_15^7 - 4*z_15^6 + 2*z_15^5 - z_15^4 + 2*z_15^3 + 6*z_15^2 - 3*z_15) 1//15*(2*z_15^7 - 5*z_15^4 - 2*z_15^3 + 2*z_15^2 - 5*z_15 + 4) 1//5*(-z_15^7 + z_15^3 - z_15^2 + 3)])
+        s2 = matspace(K[0 1//5*(2*z_15^7 - z_15^5 + 3*z_15^4 - z_15 - 1) 1//10*(-11*z_15^7 + 10*z_15^6 + 3*z_15^5 - 4*z_15^4 + 5*z_15^3 - 5*z_15^2 - 2*z_15 + 8) ; 1//5*(3*z_15^7 - 4*z_15^6 - z_15^5 + 3*z_15^4 - 3*z_15^3 + z_15^2 - z_15 - 3) 1//5*(z_15^7 - z_15^3 + z_15^2 + 2) 1//10*(-3*z_15^7 + 5*z_15^5 + 3*z_15^3 - 3*z_15^2 + 4) ; 1//15*(z_15^7 - 8*z_15^6 + 3*z_15^5 - 4*z_15^4 - z_15^3 + 7*z_15^2 - 2*z_15 - 1) 1//15*(-3*z_15^7 - 5*z_15^5 + 3*z_15^3 - 3*z_15^2 - 1) 1//5*(-z_15^7 + z_15^3 - z_15^2 + 3)])
+        s3 = matspace(K[-1 0 0 ; 0 1 0 ; 0 0 1])
+                            
+        push!(gens, s1)
+        push!(gens, s2)
+        push!(gens, s3)
+
+        W = matrix_group(gens)
+
+    elseif n == 28
+        K = QQ
```

Comment:
There is the issue with type stability here (and in some similar functions). The return type can be either `MatrixGroup{QQFieldElem, QQMatrix}` or this but with `AbsSimpleNumFieldElem` etc. I don't see a way to avoid it, but wanted to mention it in case somebody has a bright idea.

Reply:
Doesn't this religious hunt for type stability apply only for performance-critical functions? Who cares for this function?

Reply:
Yeah, and as I said, I don't see how to avoid it. QQ is a number field mathematically but not by its type.

Reply:
I just wanted to check how the performance compares, but it turns out that GAP doesn't like `rationals_as_number_field` as base field of a matrix group. So don't uses it right now. I'll look into it...

Reply:
I opened a pull request #3501 making `rationals_as_number_field` work with GAP. For GAP, there is now no difference between `QQ` or `rationals_as_number_field` on the OSCAR side. Still, there will be some performance penalty, if one works with the matrices themselves. What do you think @thofma ?

Reply:
I will ignore this for now, you can have a go once it's sorted out.